docs/user-guide/api-mediation/api-mediation-smf.md

```diff
@@ -0,0 +1,96 @@
+    ```
+
+### Using TSS
+
+If you use TSS, verify and update permission in `FACILITY` class.
+
+**Follow these steps:**
+
+1. verify user `ZWESVUSR` has read access.
+    ```      
+    TSS WHOHAS IBMFAC(IRR.RAUDITX)
+    ```    
+2. Add user `ZWESVUSR` permission to read.
+    ```
+    TSS PER(ZWESVUSR) IBMFAC(IRR.RAUDITX) ACCESS(READ)
+    ```
+   
+## SMF record configurable parameters
+
+The following list of parameters is required to set up SMF records, and the default values of these parameters can be overwritten in `zowe.yaml`. See [how to configure rauditx parameters](#configure-rauditx-parameters).
+
+| Parameter                 | Description                                                                                                                                                                                   | Type    | Default value |
+|---------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|:-------------:|
+| rauditx.fmid              | FMID of the product or component issuing the SMF record                                                                                                                                       | string  |    AZWE001    |
+| rauditx.component         | Name of the product or component issuing the SMF record                                                                                                                                       | string  |     ZOWE      |
+| rauditx.subtype           | SMF type 83 record subtype assigned to the component. For more information, see [description of subtypes](https://www.ibm.com/docs/en/zos/2.5.0?topic=records-record-type-83-security-events) | integer |       2       |
+| rauditx.event             | Event code. For more information, see [description of event codes](https://www.ibm.com/docs/en/zos/2.5.0?topic=descriptions-event-codes-event-code-qualifiers)                                | integer |       2       |
+| rauditx.qualifier.success | Event Code Qualifier for success. The value can be between 0 and 255                                                                                                                          | integer |       0       |
+| rauditx.qualifier.failed  | Event Code Qualifier for failure. The value can be between 0 and 255                                                                                                                          | integer |       1       |
+
+### Configure rauditx parameters
+
+Example of changing `rauditx.fmid` parameter. Following procedure can be applied for any of SMF record configurable parameters.
+
+**Follow these steps:**
+
+1. Open the `zowe.yaml` configuration file.
+2. Find or add the property `zowe.environments.RAUDITX_FMID` and set your desired value.
```

Comment:
Are we sure about the mapping of this value to the used property in the bean?

Reply:
Well, it was working for `apiml.security.saf.provider` untill in api-layer they added this property to `start.sh`.